plugins/in_opentelemetry/opentelemetry.c

```diff
@@ -208,6 +208,11 @@ static struct flb_config_map config_map[] = {
      "feel free to test it but please do not enable this in production " \
      "environments"
     },
+    {
+     FLB_CONFIG_MAP_BOOL, "encode_profiles_as_text", "true",
```

Comment:
if the intention is having this through the pipeline I think the property should be called `encode_profiles_as_log`

Reply:
Addressed in e67d47d